src/include/partitioning/custom_part_check_generated.rb

```diff
@@ -336,9 +336,12 @@ def check_created_partition_table(targetMap, installation)
       if (!boot_found && installation && 
           !Partitions.EfiBoot && rootdlabel=="gpt") || show_all_popups
         message = _(
-          "Warning: There is no partition of type bios_grub present.\n" +
-          "To boot from a GPT disk using grub2 such a partition is needed.\n" +
-          "\n" +
+          "Warning: There is no partition of type bios_grub present.\n" \
+          "When installing Grub2 to the MBR of a GPT formatted disc it is recommended\n" \
+          "(if Btrfs is used, then it is required) to reserve a small (around 1MB)\n" \
+          "unformatted partition of this type.\n" \
```

Comment:
Well,the sentence is taken from the bug report. And the report is precisely about having a more informative pop-up. I'll try to make the sentence shorter and more readable without losing the information the reporter wanted to have there.